packages/storage-plugin/README.md

```diff
@@ -0,0 +1,143 @@
+# Storage Plugin
+
+The purpose of Storage plugin is to create server-side file storage.
+This allows to manage various files, like application packages,
+from the client side. Only one storage may exist per server process,
+shared by all testing sessions.
+
+> [!WARNING]
+> This plugin is designed to be used with servers deployed in private networks.
+> Consider validating the setup with your security department
+> if you want to enable this plugin at a public Appium server deployment.
+
+## Installation
+
+```bash
+appium plugin install --source=npm @appium/storage-plugin
+```
+
+## Usage
+
+Add the plugin name to the list of plugins to use upon server startup:
+
+```bash
+appium --use-plugins=storage
+```
+
+By default, the plugin creates a new temporary folder where it manages uploaded files.
+
+### Environment Variables
+
+#### APPIUM_STORAGE_ROOT
+
+It is also possible to customize the repository root folder path by assigning a custom path to the
+`APPIUM_STORAGE_ROOT` environment variable upon server startup. The plugin automatically deletes the
+root folder recursively upon server process termination, unless the server is
+killed forcefully. If `APPIUM_STORAGE_ROOT` points to folder, which already exists,
+then only files managed by the plugin lifecycle are going to be deleted upon storage
+reset or upon server process termination.
+
+#### APPIUM_STORAGE_KEEP_ALL
+
+If this environment variable is set to `true`, `1` or `yes` then the plugin will always keep
+the previously uploaded storage files even after the server process is terminated.
+
+After the plugin is activated you may use the following API endpoints at your Appium server.
+These APIs are not connected to sessions and might be invoked without creating a test session.
+Such design allows to prepare the testing environment in the `before` hook prior to
+creating any driver instances.
+
+### POST /storage/add
+
+```bash
+curl -X POST --header "Content-Type: application/json" --data '{"name":"app.ipa","sha1":"ccc963411b2621335657963322890305ebe96186"}' http://127.0.0.1:4723/storage/add
+```
+
+The purpose of this command is to upload files to the remote storage partially
+to avoid excessive memory usage. In order to upload a file from a local file system
+to the Appium server's file system it is necessary to perform following steps:
+
+- Calculate [SHA1](https://en.wikipedia.org/wiki/SHA-1#:~:text=In%20cryptography%2C%20SHA%2D1%20(,rendered%20as%2040%20hexadecimal%20digits.) hash of the source file
```

Comment:
```suggestion
- Calculate <a href="https://en.wikipedia.org/wiki/SHA-1#:~:text=In%20cryptography%2C%20SHA%2D1%20(,rendered%20as%2040%20hexadecimal%20digits.">SHA1</a> hash of the source file
```

Or I think this link simply can be `https://en.wikipedia.org/wiki/SHA-1`?

Reply:
changed